Formula/radare2.rb

```diff
@@ -73,13 +86,22 @@ def install
     end
     system "make", "install"
 
+    resource("extras").stage do
+      ENV.append_path "PATH", bin
+      ENV.append_path "PKG_CONFIG_PATH", "#{lib}/pkgconfig"
+
+      system "./configure", "--prefix=#{prefix}"
+      system "make", "all"
+      system "make", "install"
+    end
+
     resource("bindings").stage do
       ENV.append_path "PATH", bin
       ENV.append_path "PKG_CONFIG_PATH", "#{lib}/pkgconfig"
 
       # Language versions.
       perl_version = `/usr/bin/perl -e 'printf "%vd", $^V;'`
-      lua_version = "5.1"
+      lua_version = `lua -e '_MAJ,_MIN=string.match(_VERSION,"^[^%d]+(%d+)%.(%d+).*$");io.write(table.concat({_MAJ,".",_MIN}))'`
```

Comment:
I don't know if this `-v` output is final. At least [`_VERSION`](www.lua.org/manual/5.2/manual.html#pdf-_VERSION) is defined in the manual.
A shorter version would be
```ruby
lua_version = `lua -e 'io.write(string.match(_VERSION, "Lua (%d+.%d+)"))'`
```
But I would be fine with either way.